project/src/main/scala/ScalastyleInspections.scala

```diff
@@ -0,0 +1,36 @@
+/*
```

Comment:
Hey, it seems to me that you duplicate this file instead of renaming it from `project/ScalastyleInspections.scala` to this _(and removing the package)_.

I checked in your branch, and indeed you have this file in the [old location](https://github.com/mwz/sonar-scala/blob/scalastyle-inspections-gen/project/ScalastyleInspections.scala) too.

Reply:
oh sorry, I'll get rid of the old file. It's tricky because even though I use this file as a template, I also want it to be compiled along with the other one so I can be sure it is correct. I also accidentally removed the package from the file, I'll fix that too.